### PR TITLE
Update the Browser section in footer

### LINKED ISSUE #9200: Update footer links from the 'Browsers' column to match those from Firefox download page
Follow up for #9089 

### Describe the problem and steps to reproduce it:
1. Open AMO stage
2. Scroll down to the footer section
3. Observe the links under the _Browser_ column

### What happened?
The column is still referencing the old Browser links used previously by the AMO footer

### What did you expect to happen?
The column should reference the links present in [Firefox download page](https://www.mozilla.org/en-US/firefox/new/) footer. Also, the column header should be named _Browsers_ instead of _Browser_

### Anything else we should know?
From https://github.com/mozilla/addons-frontend/issues/9089#issuecomment-595336205
> Yes, we should use the new links from the Firefox download page (with our UTM params added), and the header should be _Browsers_ (plural).

[current footer column]
image

[expected footer column]
image

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #9200

### CODE REVIEW

**Comment by jrbenny35** [comment]
@willdurand this might break the integration tests again

**Comment by championshuttler** [comment]
I will update those as well @jrbenny35 along with this in this PR , thanks for pointing out

**Comment by jrbenny35** [comment]
> I will update those as well @jrbenny35 along with this in this PR , thanks for pointing out

Awesome thanks! I can help you run them locally if need be.

File: src/amo/components/Footer/index.js [hunk 1]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
-                </a>
-              </li>
-              <li>
-                <a href="https://www.mozilla.org/firefox/android/?utm_source=addons.mozilla.org&utm_campaign=footer&utm_medium=referral">
-                  {i18n.gettext('Android Browser')}
+                <a
+                  className="Footer-desktop-link"
+                  href={`https://www.mozilla.org/firefox/new/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 2]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
+                      utm_medium: 'referral',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 3]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
                 </a>
               </li>
               <li>
-                <a href="https://www.mozilla.org/firefox/ios/?utm_source=addons.mozilla.org&utm_campaign=footer&utm_medium=referral">
-                  {i18n.gettext('iOS Browser')}
+                <a
+                  className="Footer-mobile-link"
+                  href={`https://www.mozilla.org/firefox/mobile/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 4]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
+                      utm_medium: 'referral',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 5]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
                 </a>
               </li>
               <li>
-                <a href="https://www.mozilla.org/firefox/desktop/?utm_source=addons.mozilla.org&utm_campaign=footer&utm_medium=referral">
-                  {i18n.gettext('Desktop Browser')}
+                <a
+                  className="Footer-reality-link"
+                  href={`https://mixedreality.mozilla.org/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 6]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
+                      utm_medium: 'referral',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 7]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
                 </a>
               </li>
               <li>
-                <a href="https://www.mozilla.org/firefox/channel/desktop/?utm_source=addons.mozilla.org&utm_campaign=footer&utm_medium=referral">
-                  {i18n.gettext('Beta, Nightly, Developer Edition')}
+                <a
+                  className="Footer-enterprise-link"
+                  href={`https://www.mozilla.org/firefox/enterprise/${makeQueryStringWithUTM(
+                    {
+                      utm_source: 'addons.mozilla.org',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: src/amo/components/Footer/index.js [hunk 8]
```diff
@@ -107,32 +107,67 @@ export class FooterBase extends React.Component {
...
+                      utm_medium: 'referral',
```
Comment (willdurand) [inline src/amo/components/Footer/index.js]: I think we can omit this, it's the default value.

File: tests/unit/amo/components/TestFooter.js [hunk 9]
```diff
@@ -115,5 +115,49 @@ describe(__filename, () => {
...
+
+    expect(root.find('.Footer-enterprise-link')).toHaveText('Enterprise');
+    expect(root.find('.Footer-enterprise-link')).toHaveProp(
+      'href',
+      `https://www.mozilla.org/firefox/enterprise/${makeQueryStringWithUTM({
+        utm_source: 'addons.mozilla.org',
+        utm_medium: 'referral',
+        utm_content: 'footer-link',
+        utm_campaign: null,
+      })}`,
```
Comment (willdurand) [inline tests/unit/amo/components/TestFooter.js]: I think we can also simplify the test cases here (by omitting the default params)

**Pull request merged**